### DISCUSSION TITLE
Displaying Bookmark Counts in a post

### DISCUSSION THREAD

**Discussion by Uranio** [dailydotdev/daily#1312] [Feature Requests]
Dear Daily.dev Team,

First off, kudos for the amazing work you're doing! I follow your community regularly, and it’s a valuable resource for my continuous learning.

I was thinking it might be interesting to add an indicator next to the bookmark icon that shows how many people have bookmarked a post. This could serve as an additional metric alongside likes. You could also consider incorporating this into the points system, giving more points to users whose posts are frequently bookmarked by others. Of course, it would be important to keep the identity of those who bookmark posts anonymous. I hope this suggestion is helpful!

**Comment by idoshamun** [comment]
Hi @Uranio,

Thanks for the suggestion! As a user what would you expect to get from knowing how many people bookmarked this post?

**Reply by Uranio** [reply]
hi @idoshamun 

thanks for the question, i'll try to explain as clearly as possible.

I use bookmarks primarily in these two ways:

To quickly save an article that I find very interesting but want to explore further later, perhaps when I have more time than during a quick 5-minute break at work.

To keep an article readily accessible because it contains information that I might need to reference frequently.

In general, both of these uses suggest that the article is potentially very valuable, as someone deemed it worth bookmarking for later review or quick access. Compared to a "like," bookmarking indicates a more active engagement with the content—it's something you intend to explore further or use regularly.

**Reply by idoshamun** [reply]
It's an interesting angle because I find the upvote much more valuable than the bookmarks. For example, I may bookmark something then give it a read and realize I don't like it

**Reply by Uranio** [reply]
I completely agree, it can definitely happen! Just like how it's easy to hit "like" without much thought... In any case, in my opinion, if a post has a lot of bookmarks, it means that many people found it worth considering. By combining this data with the number of likes, you can provide additional insights into the post's reputation.

These are just insights, but I believe that if, for example, I had a post with many bookmarks and few likes, it could mean that the user is more interested in saving that knowledge rather than just giving a like. Depending on my interests, this could give me extra information to decide whether to read it or not.

 On the other hand, if I see a post with many likes and few bookmarks, I might infer that it’s an opinion piece or news, even if the title doesn’t make it clear. I read a lot of posts about data science, and very often, there are articles that promise a lot in the title but turn out to be just opinion pieces or lists. It’s rare for someone to give a "dislike" in this context; at least in my experience, people tend to just ignore it.

 Having a bookmark can be more telling. I hope these considerations can be useful in some way!
